playwright.config.ts

```diff
@@ -67,12 +78,19 @@ const config: PlaywrightTestConfig = {
       use: {
         ...devices['Pixel 5'],
       },
+      dependencies: ['setup'],
     },
     {
       name: 'Mobile Safari',
       use: {
-        ...devices['iPhone 12'],
+        userAgent: devices['iPhone 12'].userAgent,
+        viewport: devices['iPhone 12'].viewport,
+        deviceScaleFactor: devices['iPhone 12'].deviceScaleFactor,
+        isMobile: false,
```

Comment:
Please add a comment with a link to the Issue where you got this tip discussing why we special-case mobile safari.